### PR TITLE
Add a section to configure Proctor and Sample Progman information

### PR DESCRIPTION
**Pull request by millerts**
Added a new section for configuring proctor and a progman sample configuration.

### CODE REVIEW

File: _deployment/checklist/004_deploy_to_kubernetes.md [hunk 1]
```diff
@@ -172,6 +172,24 @@ The following steps walk you through setting up the system to be available for p
...
+	* If things are working correctly this will give you a `spring_saml_metadata.xml` file 
+2. Log into OpenAM.
+	1. Under the `Common Tasks` tab and `Create SAMLv2 Providers` click the `Register Remote Service Provider`
+	2. Configure the Provider
+		1. Pick the realm you want to use.  Should be single one if you followed the provided deployment steps.
+		2. Paste the URL that you typed in on step 1 into `URL where metadata is located` textbox
+		3. You should not have to change the Circle of trust as it should be whatever you set up in the OpenAM configuration.
+		4. Click `Configure` at the top right.
+3. Verify it is configured in OpenAM
+	1. 	click the `Federation` tab
```
Comment (jeffjohnson9046) [inline _deployment/checklist/004_deploy_to_kubernetes.md]: Indentation looks off here

File: _deployment/checklist/004_deploy_to_kubernetes.md [hunk 2]
```diff
@@ -172,6 +172,24 @@ The following steps walk you through setting up the system to be available for p
...
+	2. You should see a `entity id` + `saml2`
```
Comment (jeffjohnson9046) [inline _deployment/checklist/004_deploy_to_kubernetes.md]: Example, maybe?  Like `proctorexamplesaml2`?

File: _deployment/configuration/tds_progman_configuration.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+student.TestRegistrationApplicationUrl=<CHANGE ME>
+student.testScoring.logDebug=true
+student.testScoring.logError=true
+performance.datasource.maxPoolSize=24
+performance.datasource.minPoolSize=12
+performance.logMaxTestOpportunities.enabled=false
+performance.logLatency.enabled=false
+```
+
+
```
Comment (jeffjohnson9046) [inline _deployment/configuration/tds_progman_configuration.md]: Maybe put something here about how to use the ProgMan properties after they've been created?  That is, update the `PROGMAN_LOCATOR` in the k8s yaml files or update the tomcat config (which is stored in `/etc/default/tomcat7` file for legacy and in the `setenv.sh` when running locally).   Not sure how valuable that'll be, or if this is even the right spot to tell a user how to update Proctor to use the new ProgMan properties

Comment (millerts) [reply _deployment/configuration/tds_progman_configuration.md]: I don't think this is necessarily helpful in this particular document.  This is really just to provide a sample progman configuration.  The deployment yml files call out the information suggested here.   I prefer deployment information to be included in the deployment checklist not in these documents.

File: _deployment/checklist/004_deploy_to_kubernetes.md (hunk 1 above)
Comment (millerts) [reply _deployment/checklist/004_deploy_to_kubernetes.md]: This is a github presentation problem.

File: _deployment/checklist/004_deploy_to_kubernetes.md (hunk 2 above)
Comment (millerts) [reply _deployment/checklist/004_deploy_to_kubernetes.md]: Example added

**Comment by millerts** [comment]
Closing this as this is now out of date

**Pull request closed without merging**